contracts/TridentRouter.sol

```diff
@@ -267,10 +261,12 @@ contract TridentRouter is ITridentRouter, SelfPermit, Multicall {
     ) internal {
         bento.deposit{value: token == USE_NATIVE ? amount : 0}(token, msg.sender, recipient, amount, 0);
     }
-
+    
+    /// @notice Increment helper.
     function _increment(uint256 i) internal pure returns (uint256) {
+        // Cannot realistically overflow on human timescales.
         unchecked {
-            return i + 1;
+            return i++;
```

Comment:
This doesn't work as expected since it returns the same i value instead of incrementing it. `i + 1` is fine and already optimal. I would also remove the comment since it doesn't apply for this generic function.

Reply:
I see. I suppose ++i would work here, but will revert to move this along.